### PR TITLE
fix: update naming conventions for documentation and sdks

### PR DESCRIPTION
**Pull request by wtrocki**
Content, API and UX team trying to make our API more client facing by documenting them in the console.redhat.com

For that API we cannot use internal names like "kas", "fleet" etc. as those would be confusing for users.

### CODE REVIEW

**Comment by wtrocki** [comment]
CC @jgiardino @dimakis

**Comment by wtrocki** [comment]
See SRS fleet manager PR as well: https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1135/files

**Comment by wtrocki** [comment]
I will regenerate codebase after review to avoid conflicts. 

CC @machi1990

File: openapi/kas-fleet-manager.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 openapi: 3.0.0 # need this as first line to allow some IDEs to know this is an openapi document.
 # All endpoints defined here will be discoverable by all users. If there is a need to keep some endpoints "private", add them to the "managed-services-api-private.yaml" file
 info:
-  title: Kafka Service Fleet Manager
```
Comment (machi1990) [inline openapi/kas-fleet-manager.yaml:4]: The name  `Kafka Service Fleet Manager` was a suggested name back then and this is where `kas-fleet-manager` came from. Has this been changed?

**Comment by machi1990** [comment]
> I will regenerate codebase after review to avoid conflicts.
> 
> CC @machi1990

The changes lgtm and make sense: the endpoints already have `kafkas_mgmt` in them so it only makes sense to change the title as well.
But I'd love to hear thoughts from others as well and if `Kafka Management API` is the agreed name.
/cc @pb82 @davidffrench @emmanuelbernard any thoughts on this?

File: openapi/kas-fleet-manager.yaml (hunk 1 above)
Comment (wtrocki) [reply openapi/kas-fleet-manager.yaml:4]: Yes. I agree. This is suggested name for "project name". Internal architecture and all internal communication.
For our customers and users however that might be quite cryptic term etc.

**Comment by wtrocki** [comment]
CC @jbyrne-redhat

**Review by valdar — APPROVED** [review]

**Comment by wtrocki** [comment]
@davidffrench looking for favour here to enable us to unblock epic. Rename affects nothing but API UI

**Comment by davidffrench** [comment]
Sorry, I normally miss all GitHub notifications. Always best to ping me on chat or email.

I hate to be a pain here but I am not on board with changing the name to `Kafka Management API`. I am not sure I grasp the need. The term fleet manager is becoming significantly more common in Red Hat with all new managed services.

**Comment by wtrocki** [comment]
To clarify:
We are not changing name of project etc. we are only looking to change name in OpenAPI file that has no impact to anything else than API UI. This file is dedicated to be used by end users and we want to have the same standard. 

This is to help our customers my documenting our API under some naming patterns so they understand our APIs

**Comment by davidffrench** [comment]
Thanks for the clarification @wtrocki . I agree with you that the API endpoints exposed through this openapi spec are specific to Kafka Management, so I am not opposed to this change.

The overall service has additional responsibilities such as fleet management etc.

**Comment by wtrocki** [comment]
Rebased :D

**Pull request merged**